src/linker/Linker.Steps/LinkAttributesStep.cs

```diff
@@ -36,9 +36,17 @@ IEnumerable<CustomAttribute> ProcessAttributes (XPathNavigator nav, ICustomAttri
 				TypeDefinition attributeType;
 
 				string internalAttribute = GetAttribute (iterator.Current, "internal");
-				if (internalAttribute == "RemoveAttributeInstances" && provider != null && !Annotations.IsMarked (provider)) {
-					IEnumerable<Attribute> removeAttributeInstance = new List<Attribute> { new RemoveAttributeInstancesAttribute () };
-					Context.CustomAttributes.AddInternalAttributes (provider, removeAttributeInstance);
+				if (internalAttribute != String.Empty) {
```

Comment:
Please rewrite this nested ifs into `if (condition) { .. continue; }` style

Reply:
I changed the implementation but I'm not sure if this is what you were referring